src/Processors/Merges/Algorithms/MergingSortedAlgorithm.cpp

```diff
@@ -87,15 +90,21 @@ void MergingSortedAlgorithm::consume(Input & input, size_t source_num)
     if (has_collation)
         queue_with_collation.push(cursors[source_num]);
     else
+    if (description.size() > 1)
```

Comment:
With change https://github.com/ClickHouse/ClickHouse/pull/36960/commits/61cbcbf0731ff50226e34b8102d6046f9f24e718, this will soon trigger an indentation warning ;-)